src/util/AuthUtils.js

```diff
@@ -229,7 +232,7 @@ export default class AuthUtils {
     const userObject = AuthUtils.getSavedUser();
     if (userObject && userObject.timeout && userObject.timeout > new Date().getTime()) {
       callback(userObject);
-    } else if (AuthUtils.config.ALL.authType === 'SAML' || AuthUtils.config.ALL.authType === 'NONE') {
+    } else if (AuthUtils.config.ALL.authType === 'SAML' || AuthUtils.config.ALL.authType === 'NONE' || AuthUtils.config.ALL.authType === 'XML') {
```

Comment:
Why is the XML case added here? If we're using XML authentication, then there is no callback to use to get the user... we will already have it.